upgrade.py

```diff
@@ -608,7 +608,7 @@ def replace_config(config_file, destination):
             primary_fs.unmount()
 
     prepUpgradeArgs = []
-    prepStateChanges = ['installation-uuid', 'control-domain-uuid', 'management-address-type', 'linstor-version']
+    prepStateChanges = ['installation-uuid', 'control-domain-uuid', 'management-address-type', 'linstor-version', 'xapi-version']
```

Comment:
I would rather add `xapi-version` (which could be interesting upstream) before `linstor-version` (which I'm less sure we'll manage to upstream easily)